bind_var_errors.m:035: In clause for `bind_var_in_negation':
bind_var_errors.m:035:   scope error: attempt to bind a non-local variable
bind_var_errors.m:035:   inside a negation.
bind_var_errors.m:035:   Variable `X' has instantiatedness [38;5;203m`free',[39;49m
bind_var_errors.m:035:   expected instantiatedness was [38;5;40m`unique(42)'.[39;49m
bind_var_errors.m:041: In clause for `bind_var_in_ite_cond(in)':
bind_var_errors.m:041:   scope error: attempt to bind a non-local variable
bind_var_errors.m:041:   inside the condition of an if-then-else.
bind_var_errors.m:041:   Variable `Y' has instantiatedness [38;5;203m`free',[39;49m
bind_var_errors.m:041:   expected instantiatedness was [38;5;40m`unique(42)'.[39;49m
bind_var_errors.m:050: In clause for `bind_var_in_lambda':
bind_var_errors.m:050:   in argument 1 (i.e. the predicate term) of
bind_var_errors.m:050:   higher-order predicate call:
bind_var_errors.m:050:   mode error: variable `Y' has instantiatedness [38;5;203m`free',[39;49m
bind_var_errors.m:050:   expected instantiatedness for non-local variables of
bind_var_errors.m:050:   lambda goals is [38;5;40m`ground'.[39;49m
bind_var_errors.m:055: In clause for `share_var_in_lambda(di)':
bind_var_errors.m:055:   in argument 1 of call to predicate
bind_var_errors.m:055:   `bind_var_errors.destroy'/1:
bind_var_errors.m:055:   mode error: variable `X' has instantiatedness
bind_var_errors.m:055:   [38;5;203m`ground',[39;49m
bind_var_errors.m:055:   expected instantiatedness was [38;5;40m`unique'.[39;49m
bind_var_errors.m:062: In clause for `clobber_var_in_lambda(di)':
bind_var_errors.m:062:   in argument 1 of call to predicate
bind_var_errors.m:062:   `bind_var_errors.destroy'/1:
bind_var_errors.m:062:   unique-mode error: the called procedure would clobber
bind_var_errors.m:062:   its argument, but variable `X' is still live.
For more information, recompile with `-E'.
